### PR TITLE
docs: apm-agent-dotnet 1.9

### PR DESCRIPTION
**Pull request by bmorelli25**
Update apm-agent-dotnet current version to 1.9.

### CODE REVIEW

**Review by gregkalapos — APPROVED** [review]
Thanks!

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

File: conf.yaml [hunk 1]
```diff
@@ -1118,9 +1118,9 @@ contents:
                         exclude_branches:   [ 0.7, 0.6]
                   - title:      APM .NET Agent
                     prefix:     dotnet
-                    current:    1.8
-                    branches:   [ master, 1.8 ]
-                    live:       [ master, 1.8 ]
+                    current:    1.9
+                    branches:   [ master, 1.9 ]
```
Comment (bmorelli25) [inline conf.yaml:1122]:
```suggestion
                    branches:   [ master, 1.9, 1.8 ]
```

File: conf.yaml [hunk 2]
```diff
@@ -1118,9 +1118,9 @@ contents:
...
+                    live:       [ master, 1.9 ]
```
Comment (russcam) [inline conf.yaml:1123]: I think we should keep the 1.8 branches listed in the documentation, even if they're flagged as superseded. Whilst we do encourage folks to move to the latest minor as soon as possible, being able to view docs specific to the major.minor is valuable.

File: conf.yaml (hunk 1 above)
Comment (bmorelli25) [reply conf.yaml:1122]: 1.8 shouldn't have been removed here. It's borking the build.

File: conf.yaml (hunk 2 above)
Comment (bmorelli25) [reply conf.yaml:1123]: Ha, looks like I beat you by a minute to this! Agreed, and already fixed above :)

**Pull request merged**